ui/src/components/session/SessionList.vue

```diff
@@ -56,13 +56,6 @@
           </template>
 
           <template v-slot:item.username="{ item }">
-            <v-tooltip
-              v-if="!item.authenticated"
-              bottom
-            >
-              <template #activator="{ on }" />
-              <span>Unauthorized</span>
-            </v-tooltip>
```

Comment:
Why did you remove?

Reply:
This is not used. Please check the code.